src/components/layout/header/HeaderAccount.tsx

```diff
@@ -1,38 +1,30 @@
-import { Avatar, Link } from '@mui/material';
-import { AvatarIcon } from '../../../assets/icons/AvatarIcon';
+import { Avatar, Box } from '@mui/material';
+import Link from 'next/link';
 import { useAppSelector } from '../../../store/store';
 
 export const HeaderAccount = () => {
   const profileData = useAppSelector((state) => state.profile.profileData);
 
-  return (
-    <>
-      {profileData.name ? (
-        <Avatar
-          sx={{ width: 38, height: 37 }}
-          src='https://i.imgur.com/XBQQHe9.png'
-          alt='Avatar'
-        />
-      ) : (
-        <AvatarIcon />
-      )}
-      {profileData.nickname ? (
-        <Link
-          textOverflow='ellipsis'
-          overflow='hidden'
-          whiteSpace='nowrap'
+  if (profileData.name) {
+    return (
+      <>
+        <Link href='/profilePage'>
+          <Avatar
+            sx={{ width: 38, height: 37 }}
```

Comment:
Use variables as in AvatarIcon

Reply:
AvatarIcon is svg and Avatar is MUI component. I don't use variables in Avatar as in svg. Maybe i don't understood.